samples/openthread/ncp/prj.conf

```diff
@@ -18,8 +18,10 @@ CONFIG_NET_LOG=y
 CONFIG_LOG=y
 CONFIG_NET_STATISTICS=y
 CONFIG_PRINTK=y
-CONFIG_OT_NCP_LOG_LEVEL_DBG=y
```

Comment:
Maybe leave this line here with the default value? For consistency with `overlay-logging.conf`.